samples/cpp/image_generation/image2image_concurrency.cpp

```diff
@@ -0,0 +1,68 @@
+// Copyright (C) 2023-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#include <vector>
```

Comment:
Tried to run this sample with 
```
./image2image_concurrency dreamlike_anime_1_0_ov/FP16 "A painting of a futuristic city" "A fantasy castle in the clouds" img.png
```
but got error (same for text2image_concurrency):
```
Starting to generate with prompt: 'A painting of a futuristic city'...
Starting to generate with prompt: 'A fantasy castle in the clouds'...
terminate called after throwing an instance of 'ov::Busy'
  what():  Infer Request is busy
Aborted (core dumped)
```

Reply:
No longer reproducible after the latest changes, it was related to decoder cloning